Amethyst/Events/HotKeyManager.swift

```diff
@@ -323,9 +324,15 @@ final class HotKeyManager: NSObject {
         hotKeyNameToDefaultsKey.append(["Swap focused window with main window", CommandKey.swapMain.rawValue])
         hotKeyNameToDefaultsKey.append(["Force windows to be reevaluated", CommandKey.reevaluateWindows.rawValue])
 
-        (1..<10).forEach { spaceNumber in
+        (1...10).forEach { spaceNumber in
             let name = "Throw focused window to space \(spaceNumber)"
-            hotKeyNameToDefaultsKey.append([name, "\(CommandKey.throwSpacePrefix.rawValue)-\(spaceNumber)"])
+
+            // Need to map spaceNumber 10 to throw-space-0 to preserve order of hotkeys
```

Comment:
Is there a reason just letting it be 10 instead of mapping it to 0 for the strings doesn't work? It seems like the only thing that actually needs to be 0 is the command's key in `default.amethyst`.